### ISSUE TITLE
`conda package -w` is case dependent on Windows

### ISSUE THREAD

**Issue by tadeu** [conda/conda#1713]
It should be case-insensitive, because windows filesystem is case insensitive.
E.g., when I run from the environment root:

`conda package -w Lib\site-packages\win32\win32file.pyd`

Ir finds the package, but if I run

`conda package -w lib\site-packages\win32\win32file.pyd`

it will report nothing.

**Comment by jdreaver** [comment]
I think the problem is the function [`which_path`](https://github.com/conda/conda/blob/master/conda/misc.py#L149) uses string comparisons to test equality against file paths. I think that function should use [`os.path.normpath()`](https://docs.python.org/2/library/os.path.html#os.path.normcase) on all paths to make the path components all lower-case so string comparison works. In fact, a nice path equality utility function that could be used throughout conda would be:

```python
from os.path import normpath, abspath

def paths_equal(path1, path2):
    return normpath(abspath(path1)) == normpath(abspath(path2))
```

**Comment by kalefranz** [comment]
fixed with #5308
